service/static/index.html

```diff
@@ -55,8 +55,9 @@ <h1>Recommendation REST API Service</h1>
                 <label class="control-label col-sm-2" for="recommendation_status">Active Status:</label>
                 <div class="col-sm-10">
                   <select class="form-control" id="recommendation_status">
-                      <option value="true" selected>True</option>
-                      <option value="false">False</option>
+                      <option value="True" selected>True</option>
+                      <option value="False">False</option>
+                      <option value="Any">Any</option>
```

Comment:
Any option is fine for search but this is inappropriate for Update function, we cannot update a recommendation to have "Any" status, could you please remove Any option?

Reply:
Right now my solution is when we update or create a recommendation, if the status is set to "Any" I take it as "True". 

I think this makes sense to some extent in the "create" scenario, since we consider "True" the default setting. But in "update" scenario it is indeed less reasonable. I can remove the "Any" option now.